.mvn/maven.config

```diff
@@ -0,0 +1,2 @@
+-Pconsume-incrementals
+-Pmight-produce-incrementals
```

Comment:
I wonder whether it could be managed via essentials.yml instead

Reply:
Not safely, no.